src/care_radiology/api/dicom.py

```
@@ -311,3 +335,35 @@ def parse_date(date_str):
+    for identifier in identifiers:
+        if not isinstance(identifier, dict):
+            continue
+
+        config_external_id = identifier.get("config")
+        if not config_external_id:
+            continue
+
+        try:
+            config = PatientIdentifierConfigCache.get_config(config_external_id)
+        except Exception:
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)